docs/cse/automation-service/automation-service-bridge.md

```diff
@@ -117,15 +117,15 @@ Login to Sumo Logic and create a new [installation token](/docs/manage/security/
    * `1SOAR_TOKEN1`
 1. To determine which is the correct SOAR_URL, see [Sumo Logic Endpoints by Deployment and Firewall Security](/docs/api/getting-started#sumo-logic-endpoints-by-deployment-and-firewall-security) and get the URL under the **API Endpoint** column. For example: `https://api.eu.sumologic.com/api/`
 
-And you can set this optional parameter (do not include spaces): `ALIAS`
+And you can set this optional parameter (do not include spaces and must be less than 20 characters): `ALIAS`
 
 An example of a configuration file would be:
 ```
 {
    "SOAR_URL":"API_ENDPOINT_FROM_FIREWALL_DOC_FOR_YOUR_REGION",
    "SOAR_TOKEN":"TOKEN_FROM_ADMINISTRATION_-->_SECURITY_-->_INSTALLATION TOKEN",
    "SIEM_URL":"https://YOUR_CSE_URL/sec",
-   "ALIAS":"YOUR_ALIAS_NO_SPACES"
+   "ALIAS":"YOUR_ALIAS_NO_SPACES_LESS_THAN_20_CHARACTERS"
```

Comment:
If the parameter is supposed to be less than 20 characters, this is a bad example, because it is more than 40 characters long. "ALIAS": "YOUR_ALIAS" would suffice.

Reply:
@jpipkin1 The proposed change is okay, so long as it explicitly states in the documentation that the ALIAS must be less than 20 characters